### PR TITLE
Max Server Frame Rate

### PR DESCRIPTION
**Pull request by ThatDan123**
CL: [New]  Allow admins to set a max server frame rate

### CODE REVIEW

**Comment by Bod9001** [comment]
how is the target frame ratehandled ?

**Comment by ThatDan123** [comment]
> how is the target frame ratehandled ?

https://docs.unity3d.com/ScriptReference/Application-targetFrameRate.html

File: UnityProject/Assets/Scripts/UI/Systems/AdminTools/ServerSettingsPage.cs [hunk 1]
```diff
@@ -1,31 +1,49 @@
...
 				{
 					value = 0;
 				}
-				
+
 				AdminCommandsManager.Instance.CmdChangePlayerLimit(value);
 			}
 		}
+
+		public void OnChangeFrameRate()
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/UI/Systems/AdminTools/ServerSettingsPage.cs:36]: only issue what happens if you relaog after seting the frame rate how do you get the updated value

Comment (ThatDan123) [reply UnityProject/Assets/Scripts/UI/Systems/AdminTools/ServerSettingsPage.cs:36]: line 20

Comment (Bod9001) [reply UnityProject/Assets/Scripts/UI/Systems/AdminTools/ServerSettingsPage.cs:36]: cool

**Comment by Bod9001** [comment]
What's the default frame rate?

**Comment by ThatDan123** [comment]
i'm pretty sure its uncapped at the moment

**Comment by Bod9001** [comment]
cool

**Review by Bod9001 — APPROVED** [review]

**Pull request merged**